docs/docs/reaching-out-to-user.mdx

```diff
@@ -0,0 +1,459 @@
+        dispatcher: CollectingDispatcher,
+        tracker: Tracker,
+        domain: Dict[Text, Any],
+    ) -> List[Dict[Text, Any]]:
+
+        name = next(tracker.get_latest_entity_values("PERSON"), "someone")
+        dispatcher.utter_message(f"Remember to call {name}!")
+
+        return []
+```
+
+### Cancelling Reminders
+
+To cancel a reminder that you've already scheduled, you need a custom action
+that returns the `ReminderCancelled()` event.
+
+Returning `ReminderCancelled()` cancels all the reminders that are currently scheduled.
+If you only want to cancel certain reminders, you can specify some parameters by which to narrow down the scheduled reminders:
+
+* `ReminderCancelled(intent="greet")` cancels all reminders with intent `greet`
+
+* `ReminderCancelled(entities={})` cancels all reminders with the given entities
+
+* `ReminderCancelled("...")` cancels the one unique reminder with the given name “`...`” that you supplied
+  during its creation
+
+
+For the call reminder example, you can define a custom action `action_forget_reminders` that cancels
+all reminders:
+
+```python
+class ForgetReminders(Action):
+    """Cancels all reminders."""
+
+    def name(self) -> Text:
+        return "action_forget_reminders"
+
+    async def run(
+        self, dispatcher, tracker: Tracker, domain: Dict[Text, Any]
+    ) -> List[Dict[Text, Any]]:
+
+        dispatcher.utter_message(f"Okay, I'll cancel all your reminders.")
+
+        # Cancel all reminders
+        return [ReminderCancelled()]
+```
+
+
+:::caution
+All reminders are cancelled whenever you shutdown your Rasa server.
+:::
+
+### Defining the Training Data
+
+#### Intents and Entities
+
+You'll need to define an intent that triggers setting the reminder and one that triggers cancelling the reminder.
+The `ReminderScheduled` event
+will automatically send the intent `EXTERNAL_reminder` to your bot after  the amount of time specified,
+so you also need to add this intent to your intents. You don't need to add any training examples,
+since the intent is reserved for the reminder.
+
+```
+intents:
+- intent: EXTERNAL_reminder
+- intent: ask_remind_call
+- intent: ask_forget_reminders
+```
+
+You should add NLU training examples for scheduling and cancelling the reminder:
+
+```yaml-rasa
+nlu:
+- intent: ask_remind_call
+  examples: |
+    - remind me to call John
+    - later I have to call Alan
+    - Please, remind me to call Vova
+    - please remind me to call Tanja
+    - I must not forget to call Juste
+- intent: ask_forget_reminders
+  examples: |
+    - Forget about the reminder
+    - do not remind me
+    - cancel the reminder
+    - cancel all reminders please
+```
+
+By adding `SpacyNLP` and `SpacyEntityExtractor` to your pipeline in your config file, you won't need to annotate any of the
+names in your training data, since Spacy has a `PERSON` dimension:
+
+```yaml-rasa
+pipeline:
+# other components
+- SpacyNLP
+- SpacyEntityExtractor
+    dimensions: ["PERSON"]
```

Comment:
this should probably go under a separate section called "configuration" or similar